packages/voter-dapp/src/ActiveRequests.js

```diff
@@ -51,6 +52,9 @@ import {
 } from "@umaprotocol/common";
 import { useTableStyles } from "./Styles.js";
 
+const snapshotMessage = "Sign For Snapshot";
+const snapshotHint = "Snapshotting is the act of recording the current UMA Token distribution for vote weighting. This needs to be done once before votes can be revealed for this round. The snapshot function can only be called by an externally owned account and cannot be called from another contract."
```

Comment:
```suggestion
const snapshotHint = "Snapshotting is the act of recording the current UMA Token distribution for vote weighting. This needs to be done once before votes can be revealed for this round. The snapshot function can only be called by an externally owned account."
```

I think the second half of that sentence is redundant because you're either an EOA or a contract

Reply:
yes, i added it in case someone doesn't know what an externally owned account means.

Reply:
So maybe its better to just say it cannot be called via a contract. One side effect I just realized is that 2Key contracts cannot call this. So we should warn users about that @mrice32 @chrismaree

Reply:
Yes ive tested it and updated it to work with 2key with this latest commit, the snapshot will bypass the 2key and go straight to voter contract. I will update the hint as well.